roles/kubernetes/master/templates/manifests/kube-podmaster.manifest.j2

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-podmaster
+  namespace: kube-system
+spec:
+  hostNetwork: true
+  containers:
+  - name: scheduler-elector
+    image: gcr.io/google_containers/podmaster:1.1
+    command:
+    - /podmaster
+    - --etcd-servers=http://127.0.0.1:2379
```

Comment:
Yes this is related to your first comment regarding distinct hosts for the etcd cluster in the inventory.
I'll work on it.